Godeps/Godeps.json

```diff
@@ -781,27 +822,27 @@
 		},
 		{
 			"ImportPath": "github.com/shurcooL/sanitized_anchor_name",
-			"Rev": "244f5ac324cb97e1987ef901a0081a77bfd8e845"
+			"Rev": "9a8b7d4e8f347bfa230879db9d7d4e4d9e19f962"
 		},
 		{
 			"ImportPath": "github.com/skynetservices/skydns/backends/etcd",
-			"Comment": "2.5.2d",
-			"Rev": "bb2ebadc9746f23e4a296e3cbdb8c01e956baee1"
+			"Comment": "2.5.1a",
```

Comment:
Yeah… before we spend too much time with this, let's make sure the godep save was correct… this looks weird